sapbtp-operator-charts/templates/deployment.yml

```diff
@@ -20,6 +22,9 @@ spec:
         {{- $secretTls := (include (print $.Template.BasePath "/secret-tls.yml") .) -}}
         {{- $configSha := (print $configmap $secret $secretTls) | sha256sum }}
         checksum/config: {{ $configSha }}
+        {{- if .Values.manager.annotations }}
+        {{- toYaml .Values.manager.annotations | nindent 8 }}
+        {{- end }}
```

Comment:
Kyma project has [a couple of custom annotations](https://github.com/kyma-incubator/sap-btp-service-operator/blob/0b4b0a7508c6181f58638a6a71d6427eeb9e3435/chart/templates/deployment.yml#L25-L26)